### PR TITLE
Correct README.rst

### PR DESCRIPTION
**Pull request by dlintin**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ A simple program which checks Python source files for errors.
 
 Pyflakes analyzes programs and detects various errors.  It works by
 parsing the source file, not importing it, so it is safe to use on
-modules with side effects.  It's also much faster.
+modules without any side effects.  It's also much faster.
```
Comment (krader1961) [inline README.rst:9]: What side-effects have you observed that warrant this change?

**Comment by asottile** [comment]
the text is correct as written

File: README.rst (hunk 1 above)
Comment (dlintin) [reply README.rst:9]: no side effects, that's why should be changed to "without any".

Comment (krader1961) [reply README.rst:9]: If English is your native language I hope your English teachers are performing a Picard double face-palm. If your native language is not English you should probably avoid making changes to the grammar of English text.

**Comment by sigmavirus24** [comment]
@dlintin I'm sorry for @krader1961's behaviour. It's inappropriate. 

I will say that I think this sentence could be written in a clearer way, even if this is written correctly as it is written today. @dlintin to help, we're saying that "If you have a module which (when imported) has side-effects those won't be taken into account or triggered because pyflakes parses the Abstract Syntax Tree." Does that make sense? If you're up for it, would you be up for sending a new PR to clarify that sentiment for you? It took me a couple reads of this to parse it correctly too just now because I'm particularly exhausted.

**Pull request closed without merging**